src/System.AppContext/src/System/AppContext.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Threading;
+using Windows.ApplicationModel;
+
+namespace System
+{
+    public static class AppContext
+    {
+        [Flags]
+        private enum SwitchValueState
+        {
+            HasFalseValue = 0x1,
+            HasTrueValue = 0x2,
+            HasLookedForOverride = 0x4, // Not used on .NET Native
+            UnknownValue = 0x8 // Not used on .NET Native
+        }
+
+        private static Dictionary<string, SwitchValueState> s_switchMap = new Dictionary<string, SwitchValueState>();
```

Comment:
>What we've previously agreed upon in cases like this is that folks (like me) continue to comment on the PR, and the author of the PR can then open an issue to address any PR feedback that was skipped.

Yes I agree with that approach. @dsgouda please file issues to track any source code changes as part of these porting PRs.